AirCasting/Dashboard/DashboardView.swift

```diff
@@ -51,8 +51,13 @@ struct DashboardView: View {
             //
             // Bug report was filled with Apple
             PreventCollapseView()
-            sessionTypePicker
-            if sessions.isEmpty { emptySessionsView } else { sessionListView }
+            if reorderButton.reorderIsON {
+                followingTab
+                ReorderingDashboard(viewModel: ReorderingDashboardViewModel(sessions: sessions, measurementStreamStorage: measurementStreamStorage), thresholds: Array(self.thresholds), measurementStreamStorage: measurementStreamStorage, urlProvider: urlProvider)
+            } else {
+                sessionTypePicker
+                if sessions.isEmpty { emptySessionsView } else { sessionListView }
```

Comment:
[SUGGESTION] Can we use short if in this case:
`sessions.isEmpty ? emptySessionsView : sessionListView`

Reply:
When I tried to do that I got an error about mismatching return type

Reply:
function builders magic. Maybe this will work:
```Swift
AnyView(sessions.isEmpty ? emptySessionsView : sessionListView)
```